test/TestImageFitting.cc

```diff
@@ -56,8 +56,7 @@ class ImageFittingTest : public ::testing::Test {
         _fov_info.set_rotation(rotation);
     }
 
-    void FitImage(std::vector<float> gaussian_model, std::string failed_message = "") {
-        std::string file_path = GetGeneratedFilePath(gaussian_model);
+    void FitImage(std::string file_path, std::string failed_message = "") {
```

Comment:
For consistency, the path parameter should still be a path object here (there's no need to convert it to a string at this stage).